tests/driver_sdp3x/README.md

```diff
@@ -18,4 +18,6 @@ make BOARD=... TEST_ITERATIONS=... -C tests/driver_sdp3x
 The sensor has an IRQn pin which indicates data ready on the sensor. This pin can be
 connected on the GPIO ports and configured in the variable `SDP3X_PARAM_IRQ_PIN`. This
 helps getting measurements as soon as they are ready instead of the 46ms wait (which is
-the maximum time the sensor might take the get the measurement ready).
\ No newline at end of file
+the maximum time the sensor might take the get the measurement ready).
+
+If using irq pin, instead of `USEMODULE = sdp3x`, use `USEMODULE = sdp3x_irq`.
```

Comment:
I would always use `sdp3x_irq` in the test so that code path is always compiled by CI.

Reply:
So, this is to include the periph_gpio_irq stuff in every test performed by CI, isn't it ?

Reply:
CI will just perform compile-tests here since there is no test runner equipped with such a sensor.
Using `sdp3x_irq` will make sure that all the IRQ code still compiles.

Reply:
Yeah, I understand :+1: